src/components/user/login.jsx

```diff
@@ -13,7 +13,9 @@ import { api } from "../../redux/api/api.generated";
 import ConfigLoader from "../../config-loader";
 import { displayError } from "../util/list/toast-component/display-toast";
 import localStorageKeys from "../util/local-storage-keys";
-
+import LoginSidebar from "../navigation/login-sidebar/login-sidebar";
+import AarhusLogo from "./aarhus-logo.svg";
```

Comment:
General instead of Aarhus specific

Reply:
<img width="222" alt="image" src="https://github.com/os2display/display-admin-client/assets/15377965/bfc8555c-8860-456b-a730-d941779bcbee">